src/installer/corehost/cli/apphost/static/singlefilehost_unixexports.src

```diff
@@ -0,0 +1,7 @@
+; Licensed to the .NET Foundation under one or more agreements.
+; The .NET Foundation licenses this file to you under the MIT license.
+; See the LICENSE file in the project root for more information.
+
+DllMain
+__progname
```

Comment:
Why do we need to export `__progname` and `environ` - what are these for?

(Needs comment at least.)

Reply:
This is for FreeBSD. I will add a comment. 
Without these we would be hiding exports used somewhere else. It will cause linker errors.

Reply:
I will add a comment, but if the trick with triggering DllMain from `coreclr_initialize` works we can drop all this deal about exporting DllMain.